ansible/group_vars/app.yml

```diff
@@ -3,7 +3,7 @@
 app_options:
 
   php_version:            '7.1' #5.4|5.5|5.6|7.0|7.1
-  nodejs_version:         '8'   #0.10|0.12|4|5|6|7|8
+  nodejs_version:         '8'   #0.10|0.12|4|5|6|7|8|9
```

Comment:
I have a question, I suggest that we keep the 0.10 and 0.12 available via the roles, but remove them from the comment here to avoid indicate that it is possible (even if it is), what do you think ?

Reply:
I agree.

Reply:
+1

Reply:
Added a commit for that